src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -74,11 +75,26 @@ public void HandleDatabaseRecordChange(DatabaseRecord record, long index)
             if (record == null)
                 return;
 
+            HandleSorters(record, index);
```

Comment:
I'm not sure it should be "here", it probably should be treated in the database level.

Reply:
Wasn't sure either. In general this is related to Indexes/Indexing/Querying, so I thought that it could be handled in one place.